automation/Dockerfile_s3-deploy-env

```diff
@@ -0,0 +1,13 @@
+# private base image requires docker login
+FROM balena/balena-img:6.20.26
+
+ARG DEPLOYER_UID
+ARG DEPLOYER_GID
+
+# hadolint ignore=DL3008
```

Comment:
Using a standard S3 action would avoid maintaining this image.

Reply:
Se previous answer, I think prepare.ts prepares the deflate files for upload and it's not just generic S3 uploading.